.github/workflows/ci.yml

```
@@ -0,0 +1,155 @@
+name: Erlang Library CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '**' ]
+
+env:
+  OTP_VERSION: 24.2
+  REBAR_VERSION: 3.18
+  THRIFT_VERSION: 0.14.2.1
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Setup BEAM
+        uses: erlef/setup-beam@v1.10
+        with:
+          otp-version: ${{ env.OTP_VERSION }}
+          rebar3-version: ${{ env.REBAR_VERSION }}
+      
+      - name: Cache _build
+        uses: actions/cache@v2
+        with:
+          path: _build/*/lib
+          key: ${{ runner.os }}-otp-${{ env.OTP_VERSION }}-build-${{ hashFiles('**/rebar.lock') }}
```

Остался только микродокоп по целесообразности `**/rebar.lock`. 🐌 
```suggestion
          key: ${{ runner.os }}-otp-${{ env.OTP_VERSION }}-build-${{ hashFiles('rebar.lock') }}
```